app/gzac/src/main/resources/config/form-flow/test-form-flow.json

```diff
@@ -0,0 +1,65 @@
+{
```

Comment:
Gzac? not in valtimo-core?

Reply:
That was easier for my test case. You can also include it in the valtimo app if you want.